src/backend/api/meals.js

```diff
@@ -8,7 +8,7 @@ const knex = require("../database");
 router.get("/", async (request, response) => {
   try {
     // knex syntax for selecting things.
-    const titles = await knex("meals").select("title");
+    const titles = await knex("meals").select("*");
```

Comment:
The best name is meals not titles
and you can make it like that const meals = await knex("meals")  
without select because you will select all so it is the same